examples/solara/data-viz/app.py

```diff
@@ -0,0 +1,91 @@
+"""# GPT-4 token encoder and decoder
```

Comment:
I think we can add a reference to this example from the Solara page, otherwise, people won't find it.